### PR TITLE
Add SPEC CPU2017 automation scripts

### PR DESCRIPTION
**Pull request by YashxD**
1. Added scripts and documentation to automate running SPEC CPU2017 benchmarks with C3 gem5.
2. Fix Docker base image and compiler package versions.
3. Revise README files and update to AsciiDoc formatting.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,83 @@
+= C3 gem5-based Performance Simulator
+
+[.lead]
+This is an experimental performance simulator for Cryptographic Capability Computing (C3).
+
+NOTE: If NOT running gem5 inside the provided docker, clone this repository with --recurse-submodules - otherwise, many tests will not build._
```
Comment (mdlemay) [inline README.adoc:6]: Spurious underscore at end of line.

File: prep_spec_run.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+SPEC_ROOT=${1:-"/spec2017"}
+
+echo "Build and Run SPEC becnhamarks natively"
```
Comment (mdlemay) [inline prep_spec_run.sh:4]: "benchmarks" misspelled.

**Comment by mdlemay** [comment]
I just pointed out a couple of typos. Otherwise, LGTM.

**Review by mdlemay — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (YashxD) [reply README.adoc]: Fixed

File: prep_spec_run.sh (hunk 2 above)
Comment (YashxD) [reply prep_spec_run.sh]: Fixed

**Pull request merged**